roles/confluent.control_center/tasks/main.yml

```diff
@@ -68,6 +68,14 @@
     mode: '764'
     recurse: yes
 
+- name: Create Data Directory
+  file:
+    path: "{{control_center.data_dir_path}}"
```

Comment:
Why not just make this task have:
```
path: "{{ control_center['properties']['confluent.controlcenter.data.dir'] }}"
```
and then we do not need a new variable and customers can update that property if they want in the already documented way

is recurse necessary? i am looking to remove it everywhere to improve idempotency. (that can be a separate PR)